libraries/AP_HAL/RCOutput.cpp

```diff
@@ -90,7 +94,7 @@ uint32_t AP_HAL::RCOutput::calculate_bitrate_prescaler(uint32_t timer_clock, uin
         }
     }
 
-    const uint32_t freq = timer_clock / prescaler;
+    const uint32_t freq = timer_clock / (prescaler + 1);
```

Comment:
This has changed the behaviour. We/I've spent a long time getting to a calculation that works for most people, need to not do this.

Reply:
Fixed